internal/cli/root.go

```
@@ -91,6 +92,25 @@ func newRootCommand(options *commandOptions) *cobra.Command {
+		Use:   "validate",
+		Short: "Validate the managed worker configuration",
+		Args:  cobra.NoArgs,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			workerConfig, err := config.LoadWorker(options.configPath)
+			if err != nil {
+				return err
+			}
+			if _, err := managedworker.New(workerConfig, io.Discard, io.Discard); err != nil {
+				return err
+			}
+			fmt.Fprintln(options.stdout, "worker configuration is valid")
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)